integration-test/common/src/test/java/com/tencent/rss/test/MultiStorageTest.java

```diff
@@ -48,6 +48,7 @@
 import org.junit.After;
 import org.junit.Before;
 import org.junit.BeforeClass;
+import org.junit.Ignore;
```

Comment:
unused?

Reply:
I will remove it.

Reply:
I will remove it, but checkstyle doesn't matter. It's strange.